astropy/coordinates/representation/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""
+In this module, we define the coordinate representation classes, which are
+used to represent low-level cartesian, spherical, cylindrical, and other
+coordinates.
+"""
+
+from .base import BaseDifferential, BaseRepresentation, BaseRepresentationOrDifferential
+from .base_sphere import BaseSphericalCosLatDifferential, BaseSphericalDifferential
+from .cartesian import CartesianDifferential, CartesianRepresentation
+from .cylindrical import CylindricalDifferential, CylindricalRepresentation
+from .physicsphere import PhysicsSphericalDifferential, PhysicsSphericalRepresentation
+from .radial import RadialDifferential, RadialRepresentation
+from .sphere import SphericalCosLatDifferential, SphericalDifferential, SphericalRepresentation
+from .unitsphere import (
+    UnitSphericalCosLatDifferential, UnitSphericalDifferential, UnitSphericalRepresentation)
+
+# isort: split
```

Comment:
Is the idea to import these only for backward compatibility? If so, probably good to comment this explicitly.

Reply:
SGTM.

Reply:
✅